lib/v2/screens/authentication/recover/recover_account_found/recover_account_found_screen.dart

```diff
@@ -110,53 +110,88 @@ class RecoverAccountFoundScreen extends StatelessWidget {
+                        child: FlatButtonLong(
+                          enabled: state.recoveryStatus == RecoveryStatus.READY_TO_CLAIM_ACCOUNT,
+                          title: "Claim account",
+                          onPressed: () {
+                            BlocProvider.of<RecoverAccountFoundBloc>(context).add(OnClaimAccountTap());
+                          },
                         ),
-                        SvgPicture.asset("assets/images/guardians/check_circle.svg"),
-                        const Text("Time Left Before Recovery Enabled"),
-                        Row(
-                          children: [
-                            Row(
-                              children: [
-                                Text('${state.currentRemainingTime?.hours ?? 0} ',
-                                    style: Theme.of(context).textTheme.headline5),
-                                Text(':', style: Theme.of(context).textTheme.subtitle3Opacity)
-                              ],
-                            ),
-                            Row(
-                              children: [
-                                Text('${state.currentRemainingTime?.min ?? 0} ',
-                                    style: Theme.of(context).textTheme.headline5),
-                                Text(':', style: Theme.of(context).textTheme.subtitle3Opacity)
-                              ],
-                            ),
-                            Row(
-                              children: [
-                                Text('${state.currentRemainingTime?.sec ?? 0} ',
-                                    style: Theme.of(context).textTheme.headline5),
-                              ],
-                            )
-                          ],
-                        ),
-                        state.recoveryStatus == RecoveryStatus.READY_TO_CLAIM_ACCOUNT
-                            ? Text("Account recovered " + state.userAccount)
-                            : const SizedBox.shrink(),
-                        Padding(
-                          padding: const EdgeInsets.only(top: 24),
-                          child: FlatButtonLong(
-                            enabled: state.recoveryStatus == RecoveryStatus.READY_TO_CLAIM_ACCOUNT,
-                            title: "Claim account",
-                            onPressed: () {
-                              BlocProvider.of<RecoverAccountFoundBloc>(context).add(OnClaimAccountTap());
-                            },
+                      ),
+                      body: SingleChildScrollView(
+                        child: Padding(
+                          padding: const EdgeInsets.symmetric(horizontal: horizontalEdgePadding),
+                          child: Column(
+                            children: [
+                              const SizedBox(
+                                height: 10,
+                              ),
```

Comment:
1 line instead 3?

Reply:
Thank you for the review Raul, I made it into one line (: